### PR TITLE
Automation

### PR DESCRIPTION
**Pull request by inromualdo**
[Issue](https://github.com/medic/cht-stock-monitoring-workflow/issues/5)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,20 +10,30 @@
...
     "exceljs": "^4.3.0",
+    "fraction.js": "^4.2.0",
     "fs-extra": "^11.1.0",
     "inquirer": "^8.2.5",
-    "luxon": "^3.3.0",
-    "chalk": "^4.1.0"
+    "luxon": "^3.3.0"
+  },
+  "config": {
+    "commitizen": {
```
Comment (eljhkrr) [inline package.json:35]: Add this to the contributing guidelines to ensure it's used

File: .github/workflows/release_package.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      - name: Commit CHANGELOG.md and package.json changes and create tag
+        run: |
+          git add "package.json"
+          git add "CHANGELOG.md"
+          git commit -m "chore: release ${{ env.NEW_VERSION }}"
+          git tag ${{ env.NEW_VERSION }}
+
+      # Publish version to public repository
+      - name: Publish
+        run: yarn publish --verbose --access public --tag ${{ env.RELEASE_TAG }}
```
Comment (eljhkrr) [inline .github/workflows/release_package.yml:71]: Yarn is not installed as a dependency

**Review by eljhkrr — COMMENTED** [review]
It's very commendable that you're using modern package management tools. Let's keep other users of the package in the loop using robust documentation so they can be effective contributors.

**Comment by paulpascal** [comment]
@inromualdo , is there any update on this ticket?

**Pull request merged**